packages/test/test-end-to-end-tests/package.json

```diff
@@ -31,7 +31,7 @@
     "test:realsvc:routerlicious": "cross-env fluid__test__driver=routerlicious FLUID_TEST_TIMEOUT=30000  npm run test:realsvc:run",
     "test:realsvc:routerlicious:report": "cross-env FLUID_TEST_REPORT=1 npm run test:realsvc:routerlicious",
     "test:realsvc:run": "mocha dist/test --config src/test/.mocharc.js",
-    "test:realsvc:tinylicious": "cross-env fluid__test__driver=tinylicious start-server-and-test start:tinylicious:test 35843 test:realsvc:tinylicious:run",
+    "test:realsvc:tinylicious": "cross-env fluid__test__driver=tinylicious start-server-and-test start:tinylicious:test 3000 test:realsvc:tinylicious:run",
```

Comment:
I think this is 7070 now?

Reply:
(Had to publish a new version of Tinylicious to pick up the port change.  Thx.)